openapi.yaml

```diff
@@ -9437,217 +9456,261 @@ components:
       required: true
       schema:
         type: string
+        default: application/vnd.whispir.template-v1+json
         enum:
           - application/vnd.whispir.template-v1+json
           - application/vnd.whispir.template-v1+xml
       description: Application specific mime-type.
       name: Content-Type
+      x-isContentType: true
```

Comment:
Explicitly calling out the type of header so that SDKs can conditionally handle these headers internally.